pkg/utils/utils.go

```diff
@@ -204,6 +206,15 @@ func HasDpdkDriver(pciAddr string) (bool, error) {
 			return true, nil
 		}
 	}
+
+	/* If there is a vdpa device associated with the device and it is bound to
```

Comment:
so i guess eventually we would want a VDPA code path here right ? (e.g HasVdpaDriver() ?)

Reply:
Technically, a vhost-vdpa driver *is a* dpdkDriver, so wouldn't `HasDpdkDriver` cover both cases?

Reply:
The vdpa drivers(`virtio_vdpa` `vhost_vdpa`) need to be added in `UserspaceDrivers` slice for `HasDpdkDriver` to detect them, no?

Reply:
@ahalim-intel. The vdpa drivers cannot be added to `UserspaceDrivers` because they are not pci drivers. They are vdpa-bus drivers. The driver that is used at the pci-bus level is independent of whether a vdpa device is created or not and also independent of what specific vdpa driver is used.